exercises/practice/hello-world/.meta/config.json

```diff
@@ -12,8 +12,7 @@
   ],
   "files": {
     "solution": [
-      "hello_world.cpp",
-      "hello_world.h"
+      "hello_world.cpp"
```

Comment:
I've removed the header file as the student could in this case just as easily work with just the source file. It makes things a bit simpler.